.repo-metadata.json

```diff
@@ -3,11 +3,12 @@
     "name_pretty": "Google APIs Common Protos",
     "client_documentation": "https://github.com/googleapis/python-api-common-protos",
     "issue_tracker": "https://github.com/googleapis/python-api-common-protos/issues",
-    "release_level": "ga",
+    "release_level": "stable",
     "language": "python",
     "library_type": "CORE",
     "repo": "googleapis/python-api-common-protos",
     "distribution_name": "googleapis-common-protos",
     "default_version": "",
-    "codeowner_team": ""
+    "codeowner_team": "",
+    "api_shortname": "googleapis-common-protos"
```

Comment:
This is not a GAPIC library, how is `api_shortname` determined for this case?

Reply:
Fixed.